apps/web/app/meetings/_components/Reservation/ReservationForm.tsx

```
@@ -13,20 +14,49 @@ interface ReservationFormProps {
+const addMinutes = (time: string, minutesToAdd: number): string => {
+  const parts = time.split(":");
+
+  if (parts.length !== 2) {
+    throw new Error("Invalid time format. Expected format HH:MM.");
+  }
+
+  const [hoursStr, minutesStr] = parts;
+  const hours = Number(hoursStr);
+  const minutes = Number(minutesStr);
+
+  if (isNaN(hours) || isNaN(minutes)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)